service/application/src/main/java/com/solace/maas/ep/event/management/agent/processor/CommandLogStreamingProcessor.java

```diff
@@ -50,28 +49,6 @@ public CommandLogStreamingProcessor(CommandLogsPublisher commandLogsPublisher,
     }
 
 
-    public void deleteExecutionLogFiles(List<Path> listOfExecutionLogFiles) {
-        boolean allFilesDeleted = listOfExecutionLogFiles
-                .stream()
-                .allMatch(this::deleteExecutionLogFile);
-        if (!allFilesDeleted) {
-            throw new IllegalArgumentException("Some of the execution log files were not deleted. Please check the logs");
-        }
-    }
-
-    private boolean deleteExecutionLogFile(Path path) {
-        try {
-            if (Files.exists(path)) {
-                Files.delete(path);
-            }
-        } catch (IOException e) {
-            log.warn("Error while deleting execution log at {}", path, e);
-            return false;
-        }
-        return true;
-    }
-
```

Comment:
Extracted these methods as is to `CommandManager` as these methods are better suited there